manifests/config.pp

```diff
@@ -14,7 +14,7 @@
 
       file { '/etc/apache2/mods-available/passenger.conf':
         ensure  => present,
-        content => template('passenger/passenger-enabled.erb'),
+        content => template($passenger::passenger_conf_template),
```

Comment:
This change and the change at L48 do change default behavior, though I don't know if it's a good or a breaking change.